### ISSUE TITLE
Error in F3 after PHP7 update

### ISSUE THREAD

**Issue by jrmax** [SSilence/selfoss#790]
Selfoss stops working after updating PHP7.

Error-log:
[:error] [pid 12974] [client 88.217.154.36:45195] PHP Fatal error:  Cannot use $this as parameter in /home/foo/public_html/selfoss/libs/f3/base.php on line 1185, referer: http...

It seems to be a problem with PHP7 and F3 and version 3.5.1 of F3 should fix it:  (https://github.com/bcosca/fatfree/issues/946)

**Comment by hardware** [comment]
Same error with Alpine Linux and php 7.0.9 :

Nginx logs :

```
[error] 29#0: *1 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Cannot use $this as parameter in /selfoss/libs/f3/base.php on line 1185" while reading response header from upstream, client: 172.17.0.7, server: , request: "GET / HTTP/1.0", upstream: "fastcgi://unix:/var/run/php-fpm.sock:", host: "selfoss.local"
```

**Comment by J4nsen** [comment]
For a temporary fix: updating the f3 libs with the current git master release works fine.

**Comment by hardware** [comment]
@jrmax Thank, it's work. https://github.com/hardware/selfoss/commit/56e0321619d24638e141ce85f99073aca6cc736a

**Comment by ziegenberg** [comment]
As jtojnar said, it's fixed and it works in 2.16.
@SSilence: This can be closed.
